app/reflexes/admin/orders_reflex.rb

```diff
@@ -124,9 +131,8 @@ def all_distributors_can_invoice?(orders)
       Enterprise.where(id: distributor_ids, abn: nil).empty?
     end
```

Comment:
`±` I guess this method can be removed too ☝️

Reply:
Yes, of course. Thanks for picking that up. I added a commit for that now.